### PR TITLE
feat: Added org info API

### LINKED ISSUE #120: org info API
**Is your feature request related to a problem? Please describe.**
As a user, I should be able to see the info of an org.

**Describe the solution you'd like**
1. Create an api: `[get] /api/org/(org-id)/` which return org info.
1. only authenticated users can access this API.
1. make swagger docs for the same 
1. add tests for the same

**response**
```
{
    "id" : <id>  
    "route_slug" : <route_slug>  
    "can_join_without_invite" : <can_join_without_invite>    
    "name" : <name>  
    "tagline" : <tagline>  
    "about" : <about>   
    "profile_pic" : <profile_pic>   
    "cover_pic" : <cover_pic>  
}
```

### PR DESCRIPTION
**Pull request by sahil9001**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.
This commit adds the functionality to fetch the information about any
organisation by an authorised user, so that users can check the details
before joining them.

**Type of Change:** (Delete irrelevant options)
- Code
- Testing

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
```
(envteesco) sahil@laptop:~/Desktop/Projects/teesco-backend$ python manage.py test
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
..........................................................
----------------------------------------------------------------------
Ran 58 tests in 11.920s
```

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: org/urls.py [hunk 1]
```diff
@@ -5,10 +5,9 @@
 urlpatterns = [
     path('', OrgView.as_view()),
     path('<int:org_id>/volunteer/',AddVolunteer,name='add_volunteer'),
+    path('<int:org_id>/', OrgDetailsView.as_view()),
     path('<int:org_id>/group/', GroupView.as_view()),
-    path('<int:org_id>/', EditOrg, name='edit_org'),
     path('<int:org_id>/volunteer/', AddVolunteer, name='add_volunteer'),
-    path('<int:org_id>/group/', GetGroup, name='get_group'),
```
Comment (anshsrtv) [inline org/urls.py]: Remove the add_volunteer URL ... It has been repeated two times

File: org/views.py [hunk 2]
```diff
@@ -53,6 +53,68 @@ def post(self, request):
...
+            return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+        else:
+            return Response(responses.admin_access_403, status.HTTP_403_FORBIDDEN)
+
+    @swagger_auto_schema(
+        operation_id="get_org",
+        responses={
+            '200': set_example(responses.get_org_200),
+            '400': set_example(responses.org_not_present_400),
+            '403': set_example(responses.user_unauthorized_403),
```
Comment (anshsrtv) [inline org/views.py]: It Should be  `401 Unauthorized` in place of 403.

**Review by anshsrtv — COMMENTED** [review]
Nicely done @sahil9001 :rocket: ... I have commented on some parts of code.. Please check them.

**Comment by sahil9001** [comment]
@naveennvrgup Can we merge this now?

File: org/views.py [hunk 3]
```diff
@@ -60,6 +60,68 @@ def post(self, request):
...
+        }
+    )
+    def get(self, request, org_id):
+        user = request.user
+        try:
+            org = Org.objects.get(pk=org_id)
+        except Org.DoesNotExist:
+            return Response(responses.org_not_present_400, status.HTTP_400_BAD_REQUEST)
+
+        response_body = [{
```
Comment (naveennvrgup) [inline org/views.py]:
```suggestion
        response_body = {
```

File: org/views.py [hunk 4]
```diff
@@ -60,6 +60,68 @@ def post(self, request):
...
+            "id": org.id,
+            "route_slug": org.route_slug,
+            "can_join_without_invite": org.can_join_without_invite,
+            "name": org.name,
+            "tagline": org.tagline,
+            "about": org.about,
+            "profile_pic": org.profile_pic if org.profile_pic else None,
+            "cover_pic": org.cover_pic if org.cover_pic else None
+        }]
```
Comment (naveennvrgup) [inline org/views.py]:
```suggestion
        }
```

File: org/views.py [hunk 5]
```diff
@@ -60,6 +60,68 @@ def post(self, request):
...
+
+        response_body = [{
+            "id": org.id,
+            "route_slug": org.route_slug,
+            "can_join_without_invite": org.can_join_without_invite,
+            "name": org.name,
+            "tagline": org.tagline,
+            "about": org.about,
+            "profile_pic": org.profile_pic if org.profile_pic else None,
+            "cover_pic": org.cover_pic if org.cover_pic else None
```
Comment (naveennvrgup) [inline org/views.py]:
```suggestion
            "profile_pic": request.build_absolute_uri(org.profile_pic.url) if org.profile_pic else None,
            "cover_pic": request.build_absolute_uri(org.cover_pic.url) if org.cover_pic else None
```

**Review by naveennvrgup — CHANGES_REQUESTED** [review]
1. Nice test cases :fire: 
1. Please fix conflicts

**Review by naveennvrgup — APPROVED** [review]
1. Great job @sahil9001 
1. tested locally works as expected

**Pull request merged**